Header header
float32 distance
float32 speed_av
float32 speed_max
float32 time_mv
float32 time_global
